### ISSUE TITLE
Cop idea: recommend `push` over `concat` for single element lists

### ISSUE THREAD

**Issue by vlad-pisanov** [rubocop/rubocop#11277]
I often see new Ruby devs use `concat` to add an element to a list. This cop would recommend `push` (or `<<` depending on configuration).

```ruby
# bad
list.concat [item]
# good
list.push item
list << item
```

It also happens to be about 3x faster (10M iterations here):
```
        user     system      total        real
concat  1.055298   0.037218   1.092516 (  1.092645)
push    0.368790   0.017711   0.386501 (  0.386501)
```
